src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
     <application>tech-gallery</application>
-    <version>${app.version}</version>
+    <version>1</version>
```

Comment:
Please don't specifify a hard-coded version. If necessary provide it as a Maven parameter instead.